File: problem-1/p1.cpp
// Project Euler Problem 1 solution in C++
// John Spiva

#include <iostream>

using namespace std;

int main()
{

    int sum = 0;

    for (int i = 0; i < 1000; i++)
    {
            if (i%3 == 0 || i%5 == 0)
                sum = sum + i;
    }

    cout << sum << endl;

    return 0;
}

File: problem-1/p1.js
// Project Euler Problem 1 solution in javascript
// John Spiva

var sum = 0;
for (var i = 999; i > 0; i--) {
    if ((i % 3 === 0) || (i % 5 === 0)) {
        sum += i;
    }
}
console.log(sum);
